packages/office-addin-dev-certs/src/install.ts

```diff
@@ -8,19 +8,16 @@ import { deleteCertificateFiles, uninstallCaCertificate } from "./uninstall";
 import { isCaCertificateInstalled, verifyCertificates } from "./verify";
 
 function getInstallCommand(caCertificatePath: string, machine: boolean = false): string {
-   let command: string;
-
    switch (process.platform) {
       case "win32":
-         command = `powershell Import-Certificate -CertStoreLocation cert:\\${machine ? "LocalMachine" : "CurrentUser"}\\Root '${caCertificatePath}'`;
+         return `powershell -ExecutionPolicy Bypass scripts\\install.ps1 ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
```

Comment:
Does this assume that the ExecutionPolicy will allow running the script. What happens if it is not allowed? Perhaps we need to explain how to enable it if that is the case. Yet another thing a user could run into. If this is a problem, perhaps then for the single command case you run it like earlier and otherwise you only use the script if needed.

Reply:
-ExecutionPolicy Bypass allows running scripts when the policy is disabled.

Reply:
For eg in my VM policy didn't allow me to load scripts, by adding this flag I was able to load/run scripts